indigo/distribution.yaml

```diff
@@ -12684,6 +12684,21 @@ repositories:
       url: https://github.com/turtlebot/turtlebot_create_desktop.git
       version: indigo
     status: maintained
+  turtlebot_exploration_3d:
+    doc:
+      type: git
+      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
+      version: indigo
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
```

Comment:
sure, I can use a separate release repository.

2016-08-25 17:29 GMT-04:00 Dirk Thomas <notifications@github.com>:

> In indigo/distribution.yaml
> <https://github.com/ros/rosdistro/pull/12504#discussion_r76328445>:
>
> > @@ -12684,6 +12684,21 @@ repositories:
> >        url: https://github.com/turtlebot/turtlebot_create_desktop.git
> >        version: indigo
> >      status: maintained
> > +  turtlebot_exploration_3d:
> > +    doc:
> > +      type: git
> > +      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
> > +      version: indigo
> > +    release:
> > +      tags:
> > +        release: release/indigo/{package}/{version}
> > +      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
>
> You should also use a separate release repository.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ros/rosdistro/pull/12504/files/a501576c1230317cfa52ace42d339d1a9ba47ab5#r76328445>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKFBrzQJORGn1YbVm2-HWgBgK2fHwlfSks5qjglIgaJpZM4Jtg83>
> .
>